ui.apps/src/main/content/jcr_root/apps/etoolbox-authoring-kit/depends-on/js/dependsOnUtils.js

```diff
@@ -74,8 +74,13 @@
                 return Number(value);
             case 'string':
                 return String(value);
-            case 'json':
-                return ns.parseSafe(value);
+            case 'json': {
+                const obj = ns.parseSafe(value);
+                if (obj) {
+                    return obj;
+                }
+                break;
```

Comment:
? why do we have break case here
it seems like we just need original value to be returned as it was
```suggestion
                return ns.parseSafe(value);
```
The only probable exception is that falsy value in our case should be explicitly undefined, than we just do:
```suggestion
                return ns.parseSafe(value) || undefined;
```
(or we can make it on parseSafe level)